conanfile.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os.path
+from conans import ConanFile, CMake
+from conans.tools import Git
+from conans.errors import ConanInvalidConfiguration
+
+class ImguiSfmlConan(ConanFile):
+    """ Imgui-SFML library conan file.
+
+    Options:
+
+    * shared: True or False
+      Whether to build this library shared or static
+
+    * fPIC: True or False
+      Whether or not to use -fPIC option while building
+
+    * imconfig: None or String
+      Use 'None' if you want this library to provide default
+      ImGui's imconfig.h file. Otherwise, provide a path to the
+      imconfig.h file you want to include in the build.
+
+    * imconfig_install_folder: None or String
+      Use 'None' if you want this build to include imconfig.h file,
+      otherwise use anything else.
```

Comment:
"use anything else" - better say "path to where your custom imconfig header will be installed"

Reply:
Also, there should be a way to not install custom imconfig.h (user might do it in their own CMake/conan recipe)

Reply:
> "path to where your custom imconfig header will be installed"

It's a bit confusing, actually, since:

> 24 | # If you set IMGUI_SFML_CONFIG_INSTALL_DIR, ImGui-SFML won't install your custom config, because
> 25 | # you might want to do it yourself

What does this variable do ? I think I just got it wrong.

Reply:
Okay, the comment should say "otherwise specify the directory where your imconfig will be installed"

Plus, it's not just "imconfig.h" - it's "custom user config". I'll explain it all in other "conversation" here.